src/main/java/com/kodilla/ecommercee/dao/CartDao.java

```diff
@@ -12,6 +12,8 @@
 public interface CartDao extends CrudRepository<Cart, Long> {
 
     @Override
+    Cart save(Cart cart);
```

Comment:
nieustannie uważam, że tu nic nie nadpisujemy i to jest standardowa metoda interfejsu CrudRepository więc można usunąć ten zapis, żeby nie pisać niepotrzebnego kodu. :) W przypadku finaAll() zmieniamy zwracany typ z Iterable na List i wtedy jak najbardziej nadpisujemy :)

Reply:
Możesz mieć rację, sama nie wiem jak to ma być. Można się podpytać przy okazji